### PR TITLE
Blog post on Ruby Conference Australia 2016 Gold Coast

### PR DESCRIPTION
**Pull request by ladydanger**
Hi RGSoC orga team,
Please find the PR for blog post on Ruby Conf Au tickets that were raffled off to team melbourne last year.  Thanks again for the tickets!

### CODE REVIEW

File: blog/_drafts/2016-03-20-ruby-conference-australia-2016.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+Ernie Miller’s talk was about empathy as a building block for humane development.  Humane development has at its core a simple and obvious belief that humans work with other humans to build software and this software is subsequently, used by humans.  Because of this, It’s important to cultivate empathy (i.e. how to speak the thoughts and feelings of someone who isn’t you).  Empathy makes you a better developer because you’ll have a better sense of why you’re building something and why it’s important. Your perception goes from “what would I want if I were them?” to “what would make them smile?”.  Programming is about building things and according to Miller, “If you’re empathizing well, you’ll build something people love”.  
+
+Ernie’s talk is fantastic and you can [watch a recording of his talk on empathy here](https://rubyconf.eventer.com/rubyconf-australia-2016-1489/humane-development-empathy-by-ernie-miller-1945).
+
+Below is a picture of Ernie and I - don't forget to check out his awesome t-shirt!
+
+![ernie-miller-ruby-conference-australia-2016](https://dl.dropboxusercontent.com/u/101847371/ernie-miller-and-vi-ruby-conf-au-2016.jpg)
```
Comment (alicetragedy) [inline blog/_drafts/2016-03-20-ruby-conference-australia-2016.md]: please add image to the `img/blog/2016` folder in the repo and add caption (eg. with image source/your name so we can trace it back to you)

File: blog/_drafts/2016-03-20-ruby-conference-australia-2016.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+Arne will be introducing a really exciting screencast and he's an expert in Clojure, so definitely [follow Arne on twitter](https://twitter.com/plexus).  You can also check out his [very intelligent talk called "Burn Your Idiomatic Ruby" here](https://rubyconf.eventer.com/rubyconf-australia-2016-1489/burn-your-idiomatic-ruby-by-arne-brasseur-1936).
+
+**In conclusion…**
+
+So that’s a wrap!  If you wish to [see the full set of talks, they’re online  here](https://rubyconf.eventer.com/rubyconf-australia-2016-1489).  But, before I go, this is an Australian conference right?  So I’ll leave you with a collage of party pictures and the obligatory Aussie animal pictures.
+
+![photo-collage-ruby-conference-australia](https://dl.dropboxusercontent.com/u/101847371/ruby-conf-au-collage.jpeg)
```
Comment (alicetragedy) [inline blog/_drafts/2016-03-20-ruby-conference-australia-2016.md]: please add image to the `img/blog/2016` folder

**Comment by alicetragedy** [comment]
Hey @adcopywriter, great, thanks so much for this! 
The post is great, and I like how it was structured. I've made a couple of comments on it — mainly, we need the images to be added to our repo directly (not to be linked from a dropbox or other account) so that we minimize the risk of broken links. You can find more information/a step-by-step in the [README](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/README.md#images), feel free to ping me if you need help! :sparkles:

**Comment by ladydanger** [comment]
@cyberkoi, no need - for some reason the changes I've made on my fork automatically appear in the PR, anyhow, feel free to look over the changes.

**Comment by ladydanger** [comment]
@alicetragedy - thanks for your comment!  Have made the changes/additions now, let me know if you need anything else!  Have also including a late addition to include Mila's notes on Jeff Casimir's talk b/c was waiting for her ok to include them.

**Comment by alicetragedy** [comment]
perfect. thanks @adcopywriter and @cyberkoi — I'll merge and publish today! :)

**Pull request merged**